spec/enumerables_spec.rb

```diff
@@ -0,0 +1,100 @@
+require './enumerables.rb'
+
+RSpec.describe "#my_select" do
+  describe "it should mimic the behavior of #select method" do
+    example "test 1" do
+      x = [1, 2, 3]
+      expect(x.my_select {|x| x%2 != 0}).to eql(x.select {|x| x%2 != 0})
+    end
+    example "test 2" do
+      x = ["anna", "banana", "four"]
+      expect(x.my_select {|x| x.length == 4}).to eql(x.select {|x| x.length == 4})
```

Comment:
For every example, you're comparing like a == a, it always returns true. You should instead provide input and output. This applies to all the tests.